README.md

```diff
@@ -4,24 +4,34 @@
 
 If you work on more than one feature at a time, you are guaranteed to multiply your bugs and your anxiety.
 
-## Making a plan
-
-1. **Make a drawing of your app. Simple "wireframes"**
-1. **Look at the drawing and name the HTML elements you'll need to realize your vision**
-1. **Look at the drawing and imagine using the app. What _state_ do you need to track?**
-1. **For each HTML element ask: Why do I need this? (i.e., "we need div to display the results in")**
-1. **Once we know _why_ we need each element, think about how to implement the "Why" as a "How" (i.e., `resultsEl.textContent = newResults`)**
-1. **Find all the 'events' (user clicks, form submit, on load etc) in your app. Ask one by one, "What happens when" for each of these events. Does any state change? Does any DOM update?**
-1. **Think about how to validate each of your features according to a Definition of Done. (Hint: console.log usually helps here.)**
-1. **Consider what features _depend_ on what other features. Use this dependency logic to figure out what order to complete tasks.**
-
-Additional considerations:
-
--   Ask: which of your HTML elements need to be hard coded, and which need to be dynamically generated?
--   Consider your data model.
-    -   What kinds of objects (i.e., Dogs, Friends, Todos, etc) will you need?
-    -   What are the key/value pairs?
-    -   What arrays might you need?
-    -   What needs to live in a persistence layer?
--   Is there some state we need to initialize?
--   Ask: should any of this work be abstracted into functions? (i.e., is the work complicated? can it be reused?)
+![Wireframe](wireframe.png)
+
+## HTML Elements
+
+-   text input for user question
+-   submit button to generate the fortune (add event listener)
+-   div containing the 8 ball image
+-   div to display the random fortune, set the text content
+
+## State Variables
+
+-   selectedFortune
+
+## Events
+
+1 click submit button
+_ pick a random fortune
+_ remove the input, submit button, and 8 ball img
+_ display the fortune p
+_ display a try again button
+
+2 click tempt fate button
+_ display the input, submit button, and 8 ball img
+_ remove the fortune div and tempt fate button
+
+## Fun Stuff/Stretch
+
+-   animate 8 ball on click
+-   have a generic message in the input
+
+*   do some spooky background styling
```

Comment:
yay planning! this will serve you so much as the things you're building get more and more complicated